### ISSUE TITLE
[FEATURE] TYPO3 v12 support

### ISSUE THREAD

**Issue by bh-teufels** [pixelant/pxa_product_manager#409]
Are there any plans or even a timing regarding a version for TYPO3 v12?

**Comment by bh-teufels** [comment]
Is there any information or plans about this yet?

**Comment by bh-teufels** [comment]
did you startet already an have a version to build on?
i tried it with rector but run into a lot of issues
what would be the best way to contribute where it possible to get help for some issue im not able to solve

**Comment by MattiasNilsson** [comment]
@bh-teufels No we have not started anything. What is it that you need help with then?

**Comment by nabossha** [comment]
I found that this core-bugfix for 12.4.15 is what breaks displaying a product-record in the backend:
https://github.com/typo3/typo3/commit/4b20183499

**Comment by 599media** [comment]
@nabossha Did you get it working?

**Comment by nabossha** [comment]
Yes and No. ended up stripping out the guts and changing a lot of the basic-logic on the data-model which works for a very specific use-case.

**Comment by bh-teufels** [comment]
I wonder if further development of the extension is planned, including a TYPO3 v13 version, now that it is already available.

**Comment by bh-teufels** [comment]
Would it be possible to commission an update to TYPO3 12 and TYPO3 13?
Could I get a cost estimate and a timeframe for this?

**Comment by MattiasNilsson** [comment]
@bh-teufels It is around 50 hours to get it up and running for TYPO3 v12 and v13. Let me know how you want to proceed.

**Comment by bh-teufels** [comment]
Do you already have a timeline for this extension development?

What are the options to support or accelerate the development, for example through financial contributions?

Would it be possible for you to send me an offer and a proposed timeline for this?

**Comment by MattiasNilsson** [comment]
> Do you already have a timeline for this extension development?
> 
> What are the options to support or accelerate the development, for example through financial contributions?
> 
> Would it be possible for you to send me an offer and a proposed timeline for this?

Give me an email I can send the offer and timeline to. Then we can take it from there.

**Comment by bh-teufels** [comment]
> > Do you already have a timeline for this extension development?
> > What are the options to support or accelerate the development, for example through financial contributions?
> > Would it be possible for you to send me an offer and a proposed timeline for this?
> 
> Give me an email I can send the offer and timeline to. Then we can take it from there.

I have sent an email regarding this to info@pixelant.net.

**Comment by 599media** [comment]
We might be interested too.

**Comment by CarstenHager** [comment]
Is there anything new on this topic?
Maybe we are interested too.

**Comment by bh-teufels** [comment]
Unfortunately, we have not yet received a response to our request for an offer for an update of the extension.
I sent the last request on 23.07 to info@pixelant.net

Still need a solution for TYPO3 12 & 13

**Comment by bh-teufels** [comment]
@MattiasNilsson is there anything new on this topic?

**Comment by CarstenHager** [comment]
It would be nice if the developers would comment. A financial contribution was also proposed.

**Comment by CarstenHager** [comment]
I know someone who could make the EXT compatible with V13/V14.

If anyone would like to contribute to the costs, please feel free to contact me.

ch@lab81.de
